### PR TITLE
Update third_party.rst

### PR DESCRIPTION
**Pull request by elpaso**
Following the tutorial, it seems like boilerplates configuration is needed to integrate the blog.

### CODE REVIEW

File: docs/introduction/third_party.rst [hunk 1]
```diff
@@ -42,6 +43,34 @@ Configure the image thumbnail processors in ``settings.py``::
         'easy_thumbnails.processors.filters',
     )
 
+Configure the templates that will be used by Aldryn Blog (if you configured Django CMS to use bootstrap templates, choose `bootstrap3` instead)::
```
Comment (evildmp) [inline docs/introduction/third_party.rst]: "Django CMS" > "django CMS"
"bootstrap" > "Bootstrap"

File: docs/introduction/third_party.rst [hunk 2]
```diff
@@ -42,6 +43,34 @@ Configure the image thumbnail processors in ``settings.py``::
...
+        'django.contrib.staticfiles.finders.FileSystemFinder',
+        # important! place right before django.contrib.staticfiles.finders.AppDirectoriesFinder
+        'aldryn_boilerplates.staticfile_finders.AppDirectoriesFinder',
+        'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    ]
+
+Add bolierplates loader to ``TEMPLATE_CONTEXT_PROCESSORS``::
+
+    TEMPLATE_CONTEXT_PROCESSORS = [
+        ....
```
Comment (evildmp) [inline docs/introduction/third_party.rst]: "...." > "# ..."

File: docs/introduction/third_party.rst [hunk 3]
```diff
@@ -42,6 +43,34 @@ Configure the image thumbnail processors in ``settings.py``::
...
+        'aldryn_boilerplates.context_processors.boilerplate',
+    ]
+
+Add boilerplates context processor to ``TEMPLATE_CONTEXT_PROCESSORS``::
+
+    TEMPLATE_CONTEXT_PROCESSORS = [
+        ...
```
Comment (evildmp) [inline docs/introduction/third_party.rst]: "..." > "# ..."

File: docs/introduction/third_party.rst [hunk 4]
```diff
@@ -42,6 +43,34 @@ Configure the image thumbnail processors in ``settings.py``::
...
+Add boilerplates finder to ``STATICFILES_FINDERS``::
+
+    STATICFILES_FINDERS = [
+        'django.contrib.staticfiles.finders.FileSystemFinder',
+        # important! place right before django.contrib.staticfiles.finders.AppDirectoriesFinder
+        'aldryn_boilerplates.staticfile_finders.AppDirectoriesFinder',
+        'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    ]
+
+Add bolierplates loader to ``TEMPLATE_CONTEXT_PROCESSORS``::
```
Comment (evildmp) [inline docs/introduction/third_party.rst]: Typo bolierplates

File: docs/introduction/third_party.rst [hunk 5]
```diff
@@ -42,6 +43,34 @@ Configure the image thumbnail processors in ``settings.py``::
...
+
+    TEMPLATE_CONTEXT_PROCESSORS = [
+        ....
+        'aldryn_boilerplates.context_processors.boilerplate',
+    ]
+
+Add boilerplates context processor to ``TEMPLATE_CONTEXT_PROCESSORS``::
```
Comment (evildmp) [inline docs/introduction/third_party.rst]: "Add the boilerplates..."

**Comment by evildmp** [comment]
Thanks for the documentation patch, I've made a few points inline.

**Comment by mkoistinen** [comment]
Hi @elpaso, many thanks for your contribution. Any chance you could consider @evildmp's comments and either incorporate them into your patch, or explain why we shouldn't?

**Comment by elpaso** [comment]
They are just typos, fine for me, please go ahead.

**Comment by mkoistinen** [comment]
I will merge this as is, then submit another PR to clean up.

**Pull request merged**